sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/BeamSqlCastExpression.java

```diff
@@ -77,6 +77,11 @@ public BeamSqlPrimitive evaluate(
       Row inputRow, BoundedWindow window, ImmutableMap<Integer, Object> correlateEnv) {
     SqlTypeName castOutputType = getOutputType();
     switch (castOutputType) {
+      case BOOLEAN:
```

Comment:
Random git suggestion: This little change would be a self-contained commit to support casting to boolean.

Reply:
Split this part of diff into self-contained commit. commit history on branch will be updated.